lib/facter/custom_facts/util/parser.rb

```diff
@@ -119,9 +119,9 @@ def parse_results
           cont = content.gsub(TIME, '"\1"')
 
           if Gem::Version.new(Psych::VERSION) >= Gem::Version.new('3.1.0') # Ruby 2.6+
-            YAML.safe_load(cont, permitted_classes: [Date])
+            YAML.safe_load(cont, permitted_classes: [Date], aliases: true)
           else
-            YAML.safe_load(cont, [Date])
+            YAML.safe_load(cont, [Date], aliases: true)
```

Comment:
On older ruby versions, we'll need to pass this parameter as a positional argument. Here's the older API signature https://ruby-doc.org/stdlib-2.5.0/libdoc/psych/rdoc/Psych.html#method-c-safe_load

Reply:
Ahh okay, thank you for catching that, I had no idea